packages/pdf.lua

```diff
@@ -123,6 +142,19 @@ The \command{\\pdf:destination} parameter creates a link target; it expects a
 parameter called \code{name} to uniquely identify the target. To create a link to
 that location in the document, use \code{\\pdf:link[dest=\goodbreak{}name]\{link content\}}.
 
+The \command{\\pdf:link} command accepts several options defining its border style:
+a \code{borderwidth} length setting the border width (defaults to 0, meaning no border),
+a boolean \code{borderstyle} (can be set to "underline" or "dashed", otherwise a
```

Comment:
Oops, no longer a boolean now

Reply:
Good catch, I'll fix it.

Reply:
I amended your commit with this little fix.